tasks/main.yml

```diff
@@ -35,7 +35,6 @@
     --cpu {{cpu_workers}}
     --vm {{vm_workers}}
     --hdd {{hdd_workers}}
-    --log-file {{stress_tmp_dir}}/stress-ng.log
+    --log-file {{stress_tmp_dir}}/stress-ng-{{ ansible_date_time.iso8601_basic_short }}.log
   args:
     chdir: "{{stress_tmp_dir}}"
-    creates: '/var/run/screen/S-root/*.stress'
```

Comment:
With this change it's also made possible to accidentally run multiple *simultaneous* tests.

I think it would be fair to at least warn users about this in the readme, that there are no safeguards for checking if a test is already running.

Otherwise +1 for the log timestamping.

Reply:
I added a fail instead since it is more verbose. The creates makes it non obvious if the stress test started or not.

Reply:
Good good